src/cargo.rs

```diff
@@ -554,10 +637,13 @@ pub fn rust_flags(config: &Config) -> String {
     }
     handle_llvm_flags(&mut value, config);
     if let Ok(vtemp) = env::var(RUSTFLAGS) {
+        value.push_str(vtemp.as_ref());
```

Comment:
This line should be removed as we push the vtemp value altered by the regex only

Reply:
So the then block is now empty?  Am unsure what the resultant code should look like? Do we still care about RUSTFLAGS being set in the environment?

Reply:
The block still has `value.push_str(&DEBUG_INFO.replace_all(&vtemp, " "));` in it. Remember, if RUSTFLAGS is set then the users config.toml isn't read, so we don't want to gather the rustflags in that instance

It will be:

```rust
    lazy_static! {
        static ref DEBUG_INFO: Regex = Regex::new(r#"\-C\s*debuginfo=\d"#).unwrap();
    }
    if let Ok(vtemp) = env::var(RUSTFLAGS) {
       value.push_str(&DEBUG_INFO.replace_all(&vtemp, " "));
   } else {
        let vtemp = gather_config_rust_flags(config);
        value.push_str(&DEBUG_INFO.replace_all(&vtemp, " "));
     }
```

Reply:
Ah OK -we need the new DEBUG_INFO malarkey in both the then and the else.  Thanks.